browser/src/control/Control.StatusBar.js

```diff
@@ -106,7 +106,9 @@ class StatusBar extends JSDialog.Toolbar {
 		var searchInput = L.DomUtil.get('search-input');
 		if (e.count === 0) {
 			this.enableItem('searchprev', false);
+			this.enableItem('searchprev-button', false);
```

Comment:
I think it is a workaround. We should always use "id" in this case "searchprev" only. Probably we have some bad code in the enable/disable implementation